spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+describe ReviewsController, type: :controller do
+  it 'should exist' do
+  end
+end
```

Comment:
Another empty test

Reply:
@jdmacmurtrie there's nothing in that controller to test, but if I remove the file the app yells at me. Any idea what's going on with that?